tensorbay/exception.py

```diff
@@ -149,3 +276,14 @@ def __init__(self, message: str) -> None:
 
     def __str__(self) -> str:
         return self._message
+
+
+ResponseErrorDistributor = {
```

Comment:
the key of this distributor is <str>? will it be more reasonable if key is status code?

Reply:
The key is the code of the response error.
There are errors that share the same status code. For example, the status code of InvalidParamsError, NameConflictError, and RequestParamsMissing are all 400.